gammapy/irf/effective_area.py

```diff
@@ -0,0 +1,71 @@
+#Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+from ..utils.nddata import NDDataArray, DataAxis, BinnedDataAxis
+from collections import OrderedDict
+import numpy as np
+
+class EffectiveArea2D(NDDataArray):
+    """2D effective area table
+
+    Axes: ``THETA``, ``ENERG``
```

Comment:
Yes, probably a mechanism to map FITS axis names to something else would be nice, for cases where the FITS names are cryptic.